Rakefile

```diff
@@ -27,3 +30,9 @@ task :compile do
     end
   end
 end
+
+task :clean do
```

Comment:
we can really simplify `rake clean` here because it's a mechanism built into `rake`.

```ruby
CLEAN.include(@dest)
```